### PR TITLE
Elevate missing swift ast warning to health log

### PR DESCRIPTION
**Pull request by adrian-prantl**

### CODE REVIEW

**Comment by adrian-prantl** [comment]
@swift-ci test

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp [hunk 1]
```diff
@@ -8106,10 +8106,14 @@ static swift::ModuleDecl *LoadOneModule(const SourceModule &module,
...
-               "\"Imported\" module %s via SwiftDWARFImporterDelegate "
-               "(no Swift AST or Clang module found)",
-               toplevel.AsCString());
+    HEALTH_LOG_PRINTF(
+        "Missing Swift module or Clang module found for \"%s\""
+            ", \"imported\" via SwiftDWARFImporterDelegate. Hint: %s",
+        toplevel.AsCString(),
+        swift_ast_context.GetTriple().isOSDarwin()
+            ? "Register Swift modules with the linker using -add_ast_path."
+            : "Swift modules can be wrapped in object containersusing "
```
Comment (kastiglione) [inline lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp]: missing space:

```suggestion
            : "Swift modules can be wrapped in object containers using "
```

**Review by kastiglione — APPROVED** [review]
thanks!

**Comment by adrian-prantl** [comment]
@swift-ci test

File: lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp (hunk 1 above)
Comment (adrian-prantl) [reply lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp]: thanks

**Pull request merged**